staging/src/k8s.io/apimachinery/third_party/forked/golang/reflect/deep_equal.go

```diff
@@ -100,7 +100,7 @@ func makeUsefulPanic(v reflect.Value) {
 // Tests for deep equality using reflected types. The map argument tracks
 // comparisons that have already been seen, which allows short circuiting on
 // recursive types.
-func (e Equalities) deepValueEqual(v1, v2 reflect.Value, visited map[visit]bool, depth int) bool {
+func (e Equalities) deepValueEqual(v1, v2 reflect.Value, visited map[visit]bool, implicit bool, depth int) bool {
```

Comment:
I can't see a comment changing, I'm surprised there's not a comment anywhere describing how that semantic comparison is done?

Reply:
The latest version has a comment describing the new parameter